pkg/network/ebpf/c/protocols/events.h

```diff
@@ -48,14 +53,18 @@
                     return;                                                                             \
                 }                                                                                       \
                                                                                                         \
-                long ret = bpf_perf_event_output(ctx,                                                   \
+                if (use_ring_buffer) {                                                                  \
+                    perf_ret = bpf_ringbuf_output(&name##_batch_events, batch, sizeof(batch_data_t), 0);\
+                } else {                                                                                \
+                    perf_ret = bpf_perf_event_output(ctx,                                               \
                                                  &name##_batch_events,                                  \
                                                  key.cpu,                                               \
                                                  batch,                                                 \
                                                  sizeof(batch_data_t));                                 \
```

Comment:
nit: Can you please fix the indentation?

Reply:
Done!